dask/array/tests/test_array_function.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+np = pytest.importorskip('numpy', minversion='1.16')
+
+import dask.array as da
+from dask.array.utils import assert_eq
+
+
+functions = [
+    lambda x: np.concatenate([x, x, x]),
+    lambda x: np.cov(x, x),
+    lambda x: np.dot(x, x),
+    lambda x: np.dstack(x),
+    lambda x: np.flip(x, axis=0),
+    lambda x: np.hstack(x),
+    lambda x: np.matmul(x, x),
+    lambda x: np.mean(x),
+    lambda x: np.stack([x, x]),
+    lambda x: np.sum(x),
+    lambda x: np.var(x),
+    lambda x: np.vstack(x),
+    lambda x: np.fft.fft(x.rechunk(x.shape) if isinstance(x, da.Array) else x),
+    lambda x: np.fft.fft2(x.rechunk(x.shape) if isinstance(x, da.Array) else x),
+    lambda x: np.linalg.norm(x)
+]
+
+
+notimpl_functions = [
+    lambda x: np.min_scalar_type(x),
+    lambda x: np.linalg.det(x),
+    lambda x: np.linalg.eigvals(x)
+]
+
+
+@pytest.mark.parametrize('func', functions)
```

Comment:
If `functions` is only being used once then I have a preference to just inline it here in the `parametrize` call rather than have a separate variable.  This removes one level of indirection for future readers of the code.  I find it nice to have values defined where they are used if they are only used in one place.  I accept that there are different views on this though.  This is just subjective.